projects/onto-search/src/lib/onto-search-facet/range-facet/range-slider/range-slider.component.scss

```diff
@@ -0,0 +1,109 @@
+@import '../range-facet.component';
+
+$range-width: 100% !default;
+
+$range-handle-color: $shade-10 !default;
+$range-handle-color-hover: $teal !default;
+$range-handle-size: 20px !default;
+
+$range-track-color: $shade-1 !default;
+$range-track-height: 10px !default;
+
+$range-label-color: $shade-10 !default;
+$range-label-width: 60px !default;
+
+.range-slider {
+  width: $range-width;
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  place-content: center space-around;
+  align-items: center;
+}
+
+.min-max-slider {
+  width: 100%;
+  text-align: center;
+  cursor: pointer;
+  display: flex;
+  flex-direction: row;
+  place-content: stretch center;
+  align-items: center;
+  max-height: 100%;
+}
+
+.range-values {
+  display: flex;
+  flex-direction: row;
+  place-content: space-between;
+  align-items: center;
+  margin-top: 12px;
+}
+
+.range-selecton {
+  margin-top: 12px;
+}
+
+.range-slider-range {
+  -webkit-appearance: none;
+  width: $range-width;
+  height: $range-track-height;
+  background: $range-track-color;
+  outline: none;
+  padding: 0;
+  margin: 0;
+
+  &::-webkit-slider-thumb {
```

Comment:
Will this work on browsers different than chrome? Can't autoprefixer normalize it?

Reply:
What about `::-ms-thumb`? Make a directve for this at least these are same for chrome and firefox